Dockerfile

```diff
@@ -17,6 +17,9 @@ RUN sed -i 's/#autopurge.snapRetainCount=3/autopurge.snapRetainCount=5/' /opt/zo
 # Increase the tick time so we can increase the session timeout to 5 minutes
 RUN sed -i 's/tickTime=2000/tickTime=15000/' /opt/zookeeper/conf/zoo.cfg
 
+# Update the initLimit from 10 (default) to 100
```

Comment:
Let's add a description of what changing this property actually means for how Zookeeper operates and the reason we are setting it to 100